data/json/snippets/nightmares.json

```diff
@@ -42,7 +42,12 @@
       "You have a strange dream about walking through a street from your childhood, where every surface is the same shade of solid red.  You wake up startled after noticing an equally red man down the street, moving quickly in your direction.",
-      "In your dreams you feel as though you've learned something really, really important.  You wake up in quiet distress, as you've already forgotten what that something was."
+      "In your dreams you feel as though you've learned something really, really important.  You wake up in quiet distress, as you've already forgotten what that something was.",
+      "You open your eyes and find yourself in a school hallway.  It's high school.  Students are filing past one another to reach their respective classrooms.  You begin heading for your class as well, when you distinctly hear a soft voice crying out for help.  Startled, you go into a half-sprint and rush to where the noises are coming from.  You end up in front of one of the student bathrooms.  Although the weeping seems to be echoing from the ladies' restroom, you brace yourself and barge in.  The abrupt silence catches you off guard, and of the four stalls, one seems to be painted entirely black.  You creep towards it and silently push ajar the entrance.  You find with abject terror a naked baby blown to horrific proportions wailing out his lungs.  The macabre dread deadens your heart as you realize. This was how you were birthed into this world.",
+      "You fitfully dreamed horrid, fleeting dreams for the past several days.  Each time you plunged into slumber, you always ended up in the same place.  You would be in one of the residential flats of an old burned-down building.  You rented the place due to its absurd cheapness.  The dream merely consisted of you heading to the bathroom to wash your hair over the sink.  The problem was that you could feel someone approaching in the distance.  Every night, footsteps appeared to be getting closer.  Last week, you managed to see the door to your flat opening as you headed for the bathroom.  Outside stood a man, cloaked in heavy shadow, his breathing stifled.  Four days ago, he reached the living room.  Two days ago, he stood right next to you as you entered the restroom.  His eyes were dull and resembled cracked, foggy glass.  His hands clutched a rusty knife.  Yesterday, he followed you in.  Today, he stared unblinking as you washed your hair.  Out of the corner of your eye as your cleaned yourself, you saw him scribbling something on the wall with a thick, dark red liquid.  The viscous trail of crimson shocked you awake.  It read: Tomorrow.",
```

Comment:
I would avoid insinuating that the character’s been dreaming in this manner for multiple days. I think it would be best to show each dream, placing each sequence as its own little snippet.

Reply:
I'm not sure if dreams can be dreamt in linear sequences, hence why I combined them into one. Is it possible for dreams to be coordinated?

Reply:
Indeed they can’t. a mid way solution would be to combine it as a single dream with multiple looping fazes, sort of like in the style of PT, where things get more fucked with every cycle. Also, I’d axe the little build up part that states why the character rented out the flat. It, in my view, doesn’t add much to the actual dream and just makes it longer. Good job on everything else though, some well written stuff in here.